flutter_appauth/example/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 1.0.0+1
 publish_to: 'none'
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.5.0 <3.0.0"
```

Comment:
can you make that `>=2.3.0 <3.0.0`? Support for control flow collections was actually added in 2.3

Reply:
It was available as a feature in 2.3 but as an experimental feature, it was "officially" released in Dart 2.5. If I switch it to 2.3 I will need to add `--control-flow-collections` to the analysis_options.yaml file. 

Also, since`flutter_appauth` currently depends on flutter SDK `1.12.13+hotfix.5` or greater which bundles in Dart 2.7, I can't see a situation where using this plugin with 2.3 would be possible.

Your call though, let me know how you'd like to proceed.